projects/mupdf/pdf_fuzzer.cc

```diff
@@ -51,5 +149,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
   fz_drop_context(ctx);
 
+  fprintf(stderr, "total allocation at end of execution: %" PRIu64 " bytes\n", total);
```

Comment:
It's recommended not to print anything from a fuzz target, especially inside LLVMFuzzerTestOneInput function which is executed a huge number of times.